modules/spo-binding-workloads.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-seccomp.adoc
+// * security/security_profiles_operator/spo-selinux.adoc
+
+ifeval::["{context}" == "spo-seccomp"]
+:seccomp:
+:type: seccomp
+:kind: SeccompProfile
+endif::[]
+ifeval::["{context}" == "spo-selinux"]
+:selinux:
+:type: SELinux
+:kind: SelinuxProfile
+endif::[]
+
+:_content-type: PROCEDURE
+[id="spo-binding-workloads_{context}"]
+= Binding workloads to profiles with ProfileBindings
+
+You can use the `ProfileBinding` resource to bind a security profile to the `SecurityContext` of a container.
+
+.Procedure
+
+. To bind a pod that uses a `quay.io/security-profiles-operator/test-nginx-unprivileged:1.21` image to the example `{kind}` profile, create a `ProfileBinding` object in the same namespace with the pod and the `{kind}` objects:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: security-profiles-operator.x-k8s.io/v1alpha1
+kind: ProfileBinding
+metadata:
+  namespace: my-namespace
+  name: nginx-binding
+spec:
+  profileRef:
+    kind: {kind} <1>
+    name: profile
```

Comment:
Could you add a explanation to the name: profile
The profile is hardcoded here, but it should be the name of seccompprofile/selinuxprofile

Reply:
Do you mean that the note below that currently reads "The 'kind:' variable refers to the name of the profile." should be expanded to also say "The name refers to the name of the profile."? I think that's a reasonable ask, although I suspect that the readers would figure that out.

Good fit & finish suggestion though!

Reply:
Yes, just would like to tell the "The name refers to the name of the profile". 

> s adding a similar note to e.g. the `spo-applying-profiles` might be enoug

I agree.